tests/main/parallel-install-store/task.yaml

```diff
@@ -1,31 +1,31 @@
 summary: Checks for parallel installation of snaps from the store
 
 execute: |
-    not snap install test-snapd-tools_foo 2> run.err
```

Comment:
This test needs to keep using test-snapd-tools, since the idea is that snapd reaches out to the actual store.